src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Extract.cs

```diff
@@ -8,183 +8,144 @@
 
 namespace System.IO.Compression
 {
+    /// <summary>Provides static methods for creating, extracting, and opening zip archives.</summary>
+    /// <remarks><format type="text/markdown"><![CDATA[
+    /// [!INCLUDE[remarks](~/includes/remarks/System.IO.Compression/ZipFile/ZipFile.md)]
+    /// ]]></format></remarks>
     public static partial class ZipFile
```

Comment:
Why do we have a separate copy of the xml docs for every `.cs` file that belongs to the same `partial class`? What if someone edits just one of them?

```cs
/// <summary>true</summary>
public partial class Sample { }

/// <summary>false</summary>
public partial class Sample { }
```